packages/hardhat/hardhat.config.ts

```diff
@@ -540,4 +557,283 @@ task("send", "Send ETH")
+    types.float
+  )
+  .setAction(deployGovernQueue);
+
+task("newGovernQueue:rinkeby")
+  .addParam("aclRoot", "Address that will be granted Root ACL role")
+  .addOptionalParam(
+    "governQueueFactoryAddress",
+    "Address of the govern queue factory",
+    defaultConfig.GovernQueueFactory.rinkeby
+  )
+  .addOptionalParam(
+    "resolver",
+    "Address of Celeste(IArbitrator)",
+    defaultConfig.IArbitratorCelesteAddress.rinkeby
+  )
+  .addOptionalParam(
+    "executionDelay",
+    "Execution delay for claims in seconds (default is 5 min)",
+    defaultConfig.ClaimDelay.rinkeby,
+    types.int
+  )
+  .addOptionalParam(
+    "scheduleDepositToken",
+    "Address of the schedule deposit token (default is HNYT)",
+    defaultConfig.ScheduleDeposit.rinkeby.token
+  )
+  .addOptionalParam(
+    "scheduleDepositAmount",
+    "Amount of the schedule deposit token",
+    defaultConfig.ScheduleDeposit.rinkeby.amount,
+    types.float
+  )
+  .addOptionalParam(
+    "challengeDepositToken",
+    "Address of the challenge deposit token (default is HNYT)",
+    defaultConfig.ChallengeDeposit.rinkeby.token
+  )
+  .addOptionalParam(
+    "challengeDepositAmount",
+    "Amount of the challenge deposit token",
+    defaultConfig.ChallengeDeposit.rinkeby.amount,
+    types.float
+  )
+  .setAction(deployGovernQueue);
+
+task("newGovern:gnosis")
+  .addParam(
+    "initialExecutorAddress",
+    "Address of the initial executor (should usually be the GovernQueue)"
+  )
+  .addOptionalParam(
+    "governFactoryAddress",
+    "Address of the govern factory",
+    defaultConfig.GovernFactory.xdai
+  )
+  .setAction(deployGovern);
+
+task("newGovern:rinkeby")
+  .addParam(
+    "initialExecutorAddress",
+    "Address of the initial executor (should usually be the GovernQueue)"
+  )
+  .addOptionalParam(
+    "governFactoryAddress",
+    "Address of the govern factory",
+    defaultConfig.GovernFactory.rinkeby
+  )
+  .setAction(deployGovern);
+
+task("newQuestFactory:gnosis")
+  .addOptionalParam(
+    "governAddress",
+    "Address of the govern",
+    governGnosis.address
+  )
+  .addOptionalParam(
+    "initialOwner",
+    "Initial owner of the QuestFactory (will be able to change deposits)"
+  )
+  .addOptionalParam(
+    "createDepositToken",
+    "Address of the create quest deposit (default is HNY)",
+    defaultConfig.CreateQuestDeposit.xdai.token
+  )
+  .addOptionalParam(
+    "createDepositAmount",
+    "Amount of the quest create deposit token",
+    defaultConfig.CreateQuestDeposit.xdai.amount,
+    types.float
+  )
+  .setAction(async (args, hre) => {
+    const deployResult = await deployQuestFactory(hre, args);
+    console.log(
+      "Deployed quest factory (" + hre.network.name + "):",
+      deployResult.address
+    );
+    updateContractResult(hre.network, "QuestFactory", {
+      address: deployResult.address,
+      abi: deployResult.abi,
+    });
+  });
+
+task("newQuestFactory:rinkeby")
```

Comment:
![image](https://user-images.githubusercontent.com/691510/168163457-cced9c62-af85-48ec-bbdd-295599d3041a.png)
Maybe you want setDescription for the others task too?